src/Testcontainers/Clients/TraceProgress.cs

```diff
@@ -15,7 +15,23 @@ public TraceProgress(ILogger logger)
 
     public void Report(JSONMessage value)
     {
-      this.logger.Progress(value.ProgressMessage);
+      this.LogProgressIfNotNullOrWhiteSpace(value.Stream);
+      this.LogProgressIfNotNullOrWhiteSpace(value.ProgressMessage);
+      this.LogProgressIfNotNullOrWhiteSpace(value.Status);
+
+      if (value.Error != null)
+      {
+        this.logger.LogError(value.Error.Message.Trim());
```

Comment:
Can't you use `value.ErrorMessage`? Please consider the logging concept (https://docs.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging).

Reply:
We can and I use it in few update. These properties are always equal. I also changed LogError to new extension method that I created similar way as Progress().